src/util/LocalStorageWorker.ts

```diff
@@ -18,16 +18,31 @@ import { isEventNode } from '../helpers/event';
 import { isMessage } from '../helpers/message';
 import { isSearchHistoryEntity } from '../helpers/search';
 import { EventAction, EventTreeNode } from '../models/EventAction';
-import { EventMessage } from '../models/EventMessage';
+import { EventMessage, MessageDisplayRule, MessageViewType } from '../models/EventMessage';
 import { SearchHistory } from '../stores/SearchStore';
 
 enum LocalStorageEntities {
 	PINNED_MESSAGES = 'pinnedMessages',
 	EVENTS = 'events',
 	SEARCH_HISTORY = 'search-history',
 	GRAPH_SEARCH_HISTORY = 'graph-search-history',
+	DISPLAY_RULES = 'display-rules',
 }
 class LocalStorageWorker {
+	constructor() {
+		const rules = this.getMessageDisplayRules();
```

Comment:
I think LocalStorageWorker should only save or retrieve data, any other logic is outside it's scope